server/models/Item.js

```diff
@@ -5,7 +5,32 @@ class Item extends Model {}
 
 Item.init(
   {
-    // Define your columns here
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true // Users will search for items by name, so name must be unique
+    },
+    price: {
+      type: DataTypes.NUMBER,
+      allowNull: false,
+      validate: {
+        isNumeric: true // Price has to be a number, but allow for decimals
+      }
+    },
+    description: {
+      type: DataTypes.TEXT,
+      allowNull: false
+    },
+    category: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    image: {
+      type: DataTypes.STRING,
+      validate: {
+        isUrl: true // All images in seed data are URLs, so validate image entry is a URL
```

Comment:
Perhaps add an allowNull here as we are adding new items to the inventory, the "user" may not have a selsected image yet (can always add through update)